Classes/Indexer/NodeIndexer.php

```diff
@@ -401,6 +449,72 @@ public function updateIndexAlias()
         $this->indexDriver->aliasActions($aliasActions);
     }
 
+    /**
+     * Update the main alias to allow to query all indices at once
+     */
+    public function updateMainAlias()
+    {
+        $aliasActions = [];
+        $aliasNamePrefix = $this->searchClient->getIndexNamePrefix(); // The alias name is the unprefixed index name
+
+        $indexNames = $this->indexDriver->indexesByPrefix($aliasNamePrefix);
+        $indexNames = \array_values(\array_filter($indexNames, function ($indexName) {
+            $suffix = '-' . $this->indexNamePostfix;
+            $indexNameParts = \explode('-', $indexName);
+            return substr($indexName, 0 - strlen($suffix)) === $suffix && count($indexNameParts) === 3;
```

Comment:
If we already have the indexNameParts, we could use a part instead of strlen calculations